public/app/features/query/state/queryAnalytics.ts

```diff
@@ -71,7 +71,7 @@ export function emitDataRequestEvent(datasource: DataSourceApi) {
     eventData.totalQueries = totalQueries;
     eventData.cachedQueries = cachedQueries;
 
-    const dashboard = getDashboardSrv().getCurrent();
+    const dashboard = getDashboardSrv().dashboard;
```

Comment:
Hmmm... not sure if we should change it just to make tests work. Could we add `silenceConsoleOutput` to our tests? I think it was intentional to use `getCurrent` here.

Reply:
if we pass the app to the request as well and we modify `https://github.com/grafana/grafana/blob/74167b4d44119862c1246caa3367e4f488efd15e/public/app/features/query/state/queryAnalytics.ts#L35` then there's no need for this change. this is only used for analytics, and would probably be a better solution then just silencing console output

Reply:
Done.